addons/mrp/static/src/components/wo_list_view_dropdown/wo_list_view_dropdown.scss

```diff
@@ -0,0 +1,15 @@
+
+.o_widget_mo_view_list_dropdown {
+    display: inline-block;
+    vertical-align: middle;
+    height: 100%;
+}
+
+.o_status {
+    width: 16px;
+    height: 16px;
+}
+
+.o_status_success {
+    background-color: $o-success;
+}
```

Comment:
It would be better to make it depends of a subclass in the widget. To avoid .o_status css for example to exend everywhere in odoo outside mrp :D